load-testing/audit-service/Audit-service.md

```diff
@@ -1,53 +1,49 @@
 # How to run audit service load test
 
-### Step 1:
-The load test script mocks the POSTing of some logs on `presigned-url` and `login` endpoint, we need to turn off/ comment out
-the config (only for purpose of load testing) in `kube/services/revproxy/gen3.nginx.conf/audit-service.conf` in cloud-automation.
-This should be done from within the environment.
-```
-# only allow GET requests. POSTing audit logs is only for internal use
-    limit_except GET {
-      deny all;
-    }
-```
-The above configuration denies the user to POST logs on the endpoints in audit service
-
-### Step 2:
-`gen3 kube-setup-revproxy`
+### How is load Testing performed?
+User visits the commons and logs in to the via Incommons Login or Google ir requests a PresignedURL, a message is send to AWS SQS via Fence 
```

Comment:
There are no users. The bash scripts will simulate these operations. Maybe this should be more explicit.